package NMyRpc;

message TReqSomeCall
{
    required int32 a = 1;
}

message TRspSomeCall
{
    required int32 b = 1;
}

message TReqRegularAttachments { }
message TRspRegularAttachments { }

message TReqNullAndEmptyAttachments { }
message TRspNullAndEmptyAttachments { }

message TReqDoNothing { }
message TRspDoNothing { }

message TReqCustomMessageError { }
message TRspCustomMessageError { }

message TReqNotRegistered { }
message TRspNotRegistered { }

message TReqSlowCall { }
message TRspSlowCall { }

message TReqSlowCanceledCall { }
message TRspSlowCanceledCall { }

message TReqOneWay { }
message TReqNotRegistredOneWay { }

message TReqNoReply { }
message TRspNoReply { }

